reana_cluster/backends/kubernetes/templates/deployments/job-controller-template.yaml

```diff
@@ -23,6 +23,10 @@ spec:
         env:
         - name: REANA_STORAGE_BACKEND
           value: {% if DEPLOYMENT == 'prod' %}CEPHFS{% else %}LOCAL{% endif %}
+        - name: WDB_SOCKET_SERVER
+          value: "wdb"
+        - name: WDB_NO_BROWSER_AUTO_OPEN
+          value: "True"
```

Comment:
could you please add `FLASK_DEBUG=1` in here, `server`, `workflow-monitor` and `workflow-controller`?